doc/htmldoc/synapses/connection_management.rst

```diff
@@ -256,6 +256,107 @@ must be ``True``.
+connection rule.
+
+``syn_specs`` is a dictionary of the form
+
+.. code-block:: python
+
+    {"primary": <syn_spec>,
+     "third_in": <syn_spec>,
+     "third_out": <syn_spec>}
+
+where the individual ``syn_spec`` elements follow the same rules as
+for the :py:func:`.Connect` function. Any of the three elements can be
+left out, in which case the synapse specification defaults to
+``"static_synapse"``. The ``"primary"`` synapse specification applies
+to connections from ``pre`` to ``post`` nodes, the ``"third_in"``
+specification to connections from ``pre`` to ``third`` nodes and
+the ``"third_out"`` specification to connections from ``third`` to
+``post`` nodes.
+
+tripartite_bernoulli_with_pool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each possible pair of nodes from ``A`` and ``B``, a connection is
+created with probability ``p_primary``, and these connections are
+called "primary" connections. For each primary connection, a
+third-party connection pair involving a node from ``C`` (a third
+``NodeCollection``) is created with the conditional probability
+``p_third_if_primary``. This connection pair includes a connection
+from the node from ``A`` (i.e., the source) to the node from ``C`` in
+question, and a connection from this node from ``C`` to the node from
+``B`` (i.e., the target). The node from ``C`` to connect to is chosen
+at random from a pool, a subset of the nodes in ``C``. By default,
+this pool is all of ``C``.
+
+Pool formation is controlled by parameters ``pool_type``, which can be ``"random"``
+(default) or ``"block"``, and ``pool_size`` which must be between 1
+and the size of ``C`` (default). For random pools, for each node from
+``B``, ``pool_size`` nodes from ``C`` are chosen randomly without
+replacement.
+
+For block pools, two variants exist. Let ``n(B)`` and ``n(C)`` be the number of
+nodes in ``B`` and ``C``, respectively. If ``pool_size == 1``, the
+first ``n(B)/n(C)`` nodes in ``B`` are assigned the first node in
+``C`` as their pool, the second ``n(B)/n(C)`` nodes in ``B`` the
+second node in ``C`` and so forth. In this case, ``n(B)`` must be a
+multiple of ``n(C)``. If ``pool_size > 1``, the first ``pool_size``
+elements of ``C`` are the pool for the first node in ``B``, the
+second ``pool_size`` elements of ``C`` are the pool for the second
+node in ``B`` and so forth. In this case, ``n(B) * pool_size == n(C)``
+is required.
+
+.. code-block:: python
+
+    A = nest.Create('aeif_cond_alpha_astro', 10)
+    B = nest.Create('aeif_cond_alpha_astro', 20)
+    C = nest.Create('astrocyte_lr_1994', 5)
+    conn_spec_dict = {'rule': 'tripartite_bernoulli_with_pool',
+                      'p_primary': 0.8,
+		      'p_third_if_primary': 0.5,
+                      'pool_type': 'random',
+		      'pool_size': 3}
+    syn_specs_dict = {'third_out': 'sic_connection'}
+    nest.TripartiteConnect(A, B, C, conn_spec_dict, syn_specs_dict)
+
+
+.. image:: ../static/img/tripartite_pool_type.svg
```

Comment:
I think that it would be easier to understand the figure if it had panel labels.
Also, I suggest to align it a bit more to the graphical notation in the connectivity concepts study. In #2992 the network sketches from this figure will be included in the documentation: https://doi.org/10.1371/journal.pcbi.1010086.g007
See also my other comment above on naming pre=S, post=T, third=A.

Reply:
@heplesser @jhnnsnk I have made changes according to Johanna's suggestions here. Please have a look, thanks!

I changed the figure to align with the connectivity concepts, and rearranged some text accordingly. I used a new color to distinguish the third population though. I also expanded the example code to three parts to relate to the figure. Let me know if this looks fine, or I can still improve on figure design, text arrangement, etc. Thanks.